File: dotnet/source/amp.commands/BurstRoutingCacheCommand.cs
using System.Collections.Generic;
using amp.rabbit.topology;

namespace amp.commands
{
    public class BurstRoutingCacheCommand
    {
        public IDictionary<string, RoutingInfo> NewRoutingInfo { get; set; }


        public BurstRoutingCacheCommand()
        {
            this.NewRoutingInfo = new Dictionary<string, RoutingInfo>();
        }

        public BurstRoutingCacheCommand(IDictionary<string, RoutingInfo> routingInfo)
        {
            this.NewRoutingInfo = routingInfo;
        }
    }
}

File: java/amp-rabbit/src/main/java/amp/rabbit/commands/BurstRoutingCacheCommand.java
package amp.rabbit.commands;


import java.util.Map;

import amp.rabbit.topology.RoutingInfo;


/**
 * Hello world!
 *
 */
public class BurstRoutingCacheCommand
{
    private Map<String, RoutingInfo> _routingInfo;


    public Map<String, RoutingInfo> getNewRoutingInfo() { return _routingInfo; }
    public void setNewRoutingInfo(Map<String, RoutingInfo> value) { _routingInfo = value; }


    public BurstRoutingCacheCommand() {

    }

    public BurstRoutingCacheCommand(Map<String, RoutingInfo> routingInfo) {
        _routingInfo = routingInfo;
    }
}
